bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/contentassist/CompletionProposalPopup.java

```diff
@@ -1245,9 +1245,6 @@ void displayProposals(boolean showPopup) {
 			if (showPopup) {
 				configureAndMakeVisible();
 			}
-		} else if (!fProposalShell.isVisible() && showPopup) {
```

Comment:
This was added via 78fc9f6f6566323c691f1fee1eb14de2f27e7d34.

@gayanper : could you please explain why this block was needed and how removing it would affect your original use case? Unfortunately there were no ticket reported...

Reply:
I'm not sure if I remember correctly, but I do see that the comment is not correctly phrased. But I guess there was a scenario where the documentListinser was already installed but the shell was not visible. Couldn't remember if it was due to the completion popup was canceled due to more keypress been done.